src/compass/utils/h5_helpers.py

```diff
@@ -21,7 +21,7 @@
 DATA_PATH = '/data'
 QA_PATH = '/quality_assurance'
 METADATA_PATH = '/metadata'
-
+OPERA_OPERATION_CONTACT_EMAIL = 'opera-sds-ops@jpl.nasa.gov'
```

Comment:
Since it's not directly HDF5 related, I think this is better in `helper.py`. What do you think?

Reply:
Good idea. Thanks for the suggestion :)